src/Options/index.js

```diff
@@ -520,3 +523,18 @@ export interface AuthAdapter {
   */
   enabled: ?boolean;
 }
+
+export interface LogLevelUses {
+  /* Log level used by the after hook trigger, default is 'info'.
+  :DEFAULT: 'info'
+  */
+  triggerAfterHook: ?string;
+  /* Log level used by the success before hook trigger, default is 'info.
```

Comment:
```suggestion
  /* Log level used by the Cloud Code Triggers `beforeSave`, `beforeSaveFile`, `beforeDeleteFile`, `beforeFind`, `beforeLogin` on success. Default is `info`.
```

Not sure whether LQ triggers `beforeConnect`, `beforeSubscribe` are also covered by this?